components/governance/ProposalHeader.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <header class="header">
+    <div class="header-top-column">
+      <div class="header-top-row">
+        <div>
+          <BackButton />
+        </div>
+
+        <div class="inner-status">
+          <Status :status="status" />
+        </div>
+
+        <div class="buttons">
+          <CopyButton :value="currentRoute" />
+          <Button
+            v-if="status.value === governanceStatusEnum.DEPOSITING"
+            id="deposit-btn"
+            class="action-button"
+            value="Deposit"
+            color="primary"
+            @click.native="$emit(`open-deposit-modal`)"
+          />
+          <Button
+            v-if="showVoteButton"
+            id="vote-btn"
+            class="action-button"
+            value="Vote"
+            color="primary"
+            @click.native="$emit(`open-vote-modal`)"
+          />
+        </div>
+      </div>
+
+      <div class="outer-status">
+        <Status :status="status" />
+      </div>
+    </div>
+
+    <div class="content-container">
+      <h2>{{ proposal.title }}</h2>
+
+      <div class="proposer-and-summary-container">
+        <div v-if="proposal.proposer" class="proposer">
+          Proposed By:
+          {{ proposal.proposer.address | formatAddress }}
+        </div>
+        <p class="summary">{{ proposal.summary }}</p>
+      </div>
+    </div>
+
+    <nav>
+      <ul class="page-links">
+        <li><a v-scroll-to="'#proposal-votes'" href="#">Votes</a></li>
+        <li><a v-scroll-to="'#proposal-timeline'" href="#">Timeline</a></li>
+        <li>
+          <a v-scroll-to="'#proposal-description'" href="#">Description</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import { governanceStatusEnum } from '~/common/proposal-status'
+import { formatAddress } from '~/common/address'
+
+export default {
+  name: `proposal-header`,
+  filters: {
+    formatAddress,
+  },
+  props: {
+    proposal: {
+      type: Object,
+      required: true,
+    },
+    status: {
+      type: Object,
+      required: true,
+    },
+  },
+  data: () => ({
+    governanceStatusEnum,
+  }),
+  computed: {
+    ...mapGetters([`currentNetwork`]),
+    currentRoute() {
+      return location.href
+    },
+    showVoteButton() {
+      // when the proposal is a Treasury proposal we won't show the Vote button
+      // for all Polkadot proposals we display the Vote button except for treasuries
```

Comment:
polka-no

Reply:
Missed thar. Good eye

Reply:
ouch. Removing

Reply:
We should also actually remove the `isPolkadot` check  in `ImportSeedStep` and the `incorrectPolkadotSeed` from `FormMessage`. There are a couple of polka-no left around, as I see

Reply:
Done